### DISCUSSION TITLE
elaborate on post-css config

### DISCUSSION THREAD

**Discussion by maboesanman** [vuejs/vue-cli#6873] [Q&A]
I'm trying to understand the output of `vue-cli-service inspect` better.

specifically this:

```js
{
  loader: 'postcss-loader',
  options: {
    sourceMap: false,
    plugins: [
      function () { /* omitted long function */ }
    ]
  }
}
```

what is the content of the ommitted function, and where does it come from?

**Comment by Viggo95** [comment]
For some reason, vue inspect doesn't show all webpack configuration information

**You can print the specific configuration yourself here**

https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/commands/build/index.js#L200
